xknx/io/tunnel.py

```diff
@@ -159,7 +159,7 @@ async def _reconnect(self) -> None:
             self.transport.stop()
         await asyncio.sleep(self.auto_reconnect_wait)
         if await self.connect():
-            logger.info("Successfully reconnected to KNX bus.")
+            logger.info("successfully reconnected to KNX bus.")
```

Comment:
```suggestion
            logger.info("Successfully reconnected to KNX bus.")
```
Why is this changed to lower case?